metadata-ingestion/src/datahub/ingestion/source/sql/athena.py

```diff
@@ -33,10 +33,22 @@ def get_sql_alchemy_url(self):
 
 
 class AthenaSource(SQLAlchemySource):
+    config: AthenaConfig
```

Comment:
Delete this. The `config` is already an instance member of SQLAlchemySource, and will get correctly initialized via lines 44->39.